### PR TITLE
📖 [docs] : corrected layout of manually_migration_guide_gov3_to_gov4.md

### PR DESCRIPTION
**Pull request by swastik959**
* the markdown formatting of the file manually_migration_guide_gov3_to_gov4.md was incorrect hence leading to a bad layout
especially in the last section .

* it fixes the issue #3167

* feel free to point out any discrepancy

### CODE REVIEW

**Review by everettraven — CHANGES_REQUESTED** [review]
@swastik959 Thanks for your contribution! I have a couple suggestions:

File: docs/book/src/migration/manually_migration_guide_gov3_to_gov4.md [hunk 1]
```diff
@@ -135,19 +135,20 @@ Note that if your project has multiple groups (`multigroup:true`) then the above
...
   kind: Captain
   path: sigs.k8s.io/kubebuilder/testdata/project-v4/apis/crew/v1
-```
 
-After Update:
 
-```
+**After Update:**
+
+
```
[the remark this answers is not in the document]
Comment (swastik959) [reply docs/book/src/migration/manually_migration_guide_gov3_to_gov4.md]: @everettraven 
thanks for the feedback 
you can check it now

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

Could you please sign the CLA for we move forward?

**Comment by swastik959** [comment]
> /lgtm
> 
> Could you please sign the CLA for we move forward?

@camilamacedo86 I have signed it please guide me through further process

**Comment by swastik959** [comment]
> @swastik959: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name | Commit | Details | Required | Rerun command
> --- | --- | --- | --- | ---
> pull-kubebuilder-e2e-k8s-1-25-3 | 9b697130099db14d0aca0e0489df652bb6d2c631 | [link](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder/3176/pull-kubebuilder-e2e-k8s-1-25-3/1616607375442055168) | true | `/test pull-kubebuilder-e2e-k8s-1-25-3`
> 
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes-sigs&repo=kubebuilder&pr=3176). [Your PR dashboard](https://prow.k8s.io/pr?query=is%3Apr%20state%3Aopen%20author%3Aswastik959). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes-sigs/kubebuilder/issues?q=is:issue+is:open) when you hit one in your PR.
> 
> <details>
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
> </details>
> 

@camilamacedo86  this test case failed can you please look into it

**Pull request merged**